src/js/views/globals/app-nav/app-nav.scss

```diff
@@ -44,52 +44,117 @@ $menu-hover-color: #375467;
   vertical-align: -0.125em;
 }
 
-.app-nav__title,
-.app-nav__search {
+.app-nav__title {
   font-weight: 700;
   margin-top: 20px;
-  padding: 4px 16px;
+  opacity: 0.5;
+  padding: 4px 16px 8px;
 }
 
 .app-nav__search {
+  cursor: pointer;
+  font-weight: 600;
+  margin-top: 20px;
+  opacity: 0.5;
+  padding: 8px 16px;
+
   .icon {
-    font-size: 12px;
-    margin-right: 8px;
+    font-size: 20px;
+    margin-right: 16px;
     vertical-align: -0.0825em;
+    width: 1em;
   }
 
   &:hover, &.is-active {
     background-color: $menu-hover-color;
     color: $white;
+    opacity: 1;
   }
 }
 
 .app-nav__link {
   cursor: pointer;
-  display: block;
-  padding: 4px 16px;
+  font-weight: 600;
+  opacity: 0.5;
+  padding: 8px 16px;
+  position: relative;
 
   &:hover, &.is-selected {
     background-color: $menu-hover-color;
     color: $white;
+    opacity: 1;
+
+    &::before {
+      background-color: color(light_blue);
+      bottom: 0;
+      content: '';
+      left: 0;
+      position: absolute;
+      top: 0;
+      width: 4px;
+    }
```

Comment:
This adds the light blue left border for any nav links that are hovered or currently selected.

![Screenshot 2023-01-17 at 2 03 43 PM](https://user-images.githubusercontent.com/35355575/213000342-efec2dd7-8734-402e-8e6a-9851298567b5.png)

Doing a simple `border-left: 4px solid color(light_blue)` caused the layout for each nav item to jump when hovering. That's why I went the `::before` route instead.